pages/pipelines/hosted_agents/overview.md

```diff
@@ -1,20 +1,21 @@
 # What are hosted agents
 
-Buildkite hosted agents provides a fully-managed platform on which you can run your agents.
+Buildkite hosted agents provides a fully-managed platform on which you can run your pipelines.
```

Comment:
need to stay as agents, pipelines spin up jobs that run across different agents, so a single pipeline could actually be run on both hosted and non-hosted agents, it's the agents that run on the compute not the pipeline.